vocabularies/Common.md

```diff
@@ -387,31 +387,23 @@ Property|Type|Description
 [ShareAction](Common.xml#L1131)|[QualifiedName?](#QualifiedName)|Action that shares a draft document with other users and restricts access to the listed users in their specified roles<br><p>The action is bound to the draft document root node and has the signature of <a href="#Template_ShareAction"><code>Template_ShareAction</code></a>. It restricts access to the listed users in their specified roles.</p> <p>If this action is present, the client can receive notifications about changes to the collaborative draft by opening a web socket connection at the <a href="#WebSocketBaseURL"><code>WebSocketBaseURL</code></a> followed by URL parameters</p> <ul> <li><code>relatedService</code> = base URL (relative to server root) of the OData service of the app</li> <li><code>draft</code> = draft UUID.</li> </ul> 
 
 <a name="DraftNodeType"></a>
-## [DraftNodeType](Common.xml#L1232)
-
-
-**Derived Types:**
-- [DraftRootType](#DraftRootType)
-
-Property|Type|Description
-:-------|:---|:----------
-[PreparationAction](Common.xml#L1233) *(Deprecated)*|[QualifiedName?](#QualifiedName)|Preparation is always called on the draft root node
```

Comment:
PreparationAction and ValidationFunction are removed here but not in json or xml?